### PR TITLE
Added Peacock TV to services

### PR DESCRIPTION
**Pull request by sandervankasteel**

### CODE REVIEW

File: services/peacock_tv.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
...
...-2c1.793-1.586 1.793-1.586 4.188-3.375 2.63-1.98 5.214-3.958 7.75-6.062 2.062-1.563 2.062-1.563 4.062-1.563v-2h3v-2c18.062-9.351 35.647-16.706 56-19l3.106-.386c32.078-3.13 66.565 7.2 90.894 28.386z" fill="#11a943"/><path d="M246.576 586.114h1l1 446 5-1c-1 2-1 2-4.062 3.126l-2.938.875c-1.668-1.669-1.124-3.697-1.133-5.925l.007-2.158c0-.754-.003-1.507-.004-2.284-.004-2.557 0-5.115.003-7.672 0-1.84-.003-3.678-.005-5.517-.004-5.08-.002-10.158.001-15.237.002-5.468 0-10.936-.004-16.404-.004-10.721-.003-21.442 0-32.164.002-8.71.003-17.42 0-26.13V910.337c-.004-22.92 0-45.839.004-68.759.005-20.938.004-41.877 0-62.816-.005-23.504-.007-47.007-.004-70.511v-7.513l.001-3.734c0-8.703 0-17.407-.003-26.111-.003-10.601-.002-21.202.003-31.803.003-5.414.004-10.827 0-16.241-.003-4.949 0-9.898.004-14.846.001-1.798 0-3.595-.002-5.393-.003-2.429 0-4.858.005-7.287l-.007-2.151c.02-4.83.02-4.83 1.133-7.058z" fill="#2c2d26"/><path d="m192.576 217.114 2 1-37 37-2-1c2.207-2.376 4.416-4.75 6.625-7.125l1.871-2.015a994.778 994.778 0 0 1 5.504-5.86l1.73-1.83c1.382-1.43 2.822-2.806 4.27-4.17h2v-2c1.375-1.625 1.375-1.625 3-3h2v-2c1.68-1.656 1.68-1.656 3.875-3.5 2.115-1.789 4.165-3.54 6.125-5.5z" fill="#21201d"/></svg>
```
Comment (Sergey-Lyapin) [inline services/peacock_tv.yml:6]: SVG must be square in shape.
Please use this SVG:
```
<svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="-233 0 1964 1964"><path d="M519.7 101.8h6.2c20.7 0 41.1.6 61.7 3.3l3.7.5a322.8 322.8 0 0 1 43.2 8c47.3 10.3 105.3 25.3 144 55.5v2l2.5.7a62 62 0 0 1 12.4 6.7l2.5 1.6c12.8 8.3 25 17.1 36.8 26.8l11.4 9.2c12.3 9.6 12.3 9.6 15.4 13.5a37 37 0 0 0 5.6 5.6c5.3 4.5 10.2 9.5 15 14.5a611.3 611.3 0 0 0 4 4c9 9 17.3 18.7 25.5 28.4l2.3 2.8c50.7 61 50.7 61 50.7 76.2l2 1c5.2 7.3 9 15.7 13 23.7a1540.5 1540.5 0 0 0 3 6.2c11.7 23.3 11.7 23.3 13 31.1h2c28 71.8 46.2 144.9 46.2 222.3v3.6c0 45.4-5 91.7-19.2 135.1l-1.1 3.3c-5 15.4-10.5 30.6-16.7 45.5l-.8 1.9a346.3 346.3 0 0 1-11.4 24.3h-2l-.5 1.6a84 84 0 0 1-8 17.5l-1.7 2.8a339.7 339.7 0 0 1-14.8 23.1l-1.5 2-9 13a600 600 0 0 0-4.2 5.8 161 161 0 0 1-15 18.6 176 176 0 0 0-8.5 10.3 217 217 0 0 1-8.8 10.3l-1.4 1.5c-10.4 11.5-10.4 11.5-16 16.1a47.3 47.3 0 0 0-5 5.2 96.5 96.5 0 0 1-13.6 13.2h-2v2c-1.6 1.4-1.6 1.4-3.8 3-3.8 2.9-7.3 6-10.9 9.3-6 5.6-12.5 10.5-19.3 15.2l-8.8 6.4-7 5a398 398 0 0 0-7.5 5.6c-11 8.1-22.8 14.8-34.6 21.7a673.5 673.5 0 0 0-8.4 5 330.7 330.7 0 0 1-62.2 28.5l-15 5.6a850 850 0 0 1-27.5 10.1 2134 2134 0 0 0-48 17.6 1789 1789 0 0 1-61 22.2c-5.8 2-11.4 4.1-17 6.4-8 3.2-16 6.2-24.2 9.2l-2.4.9a1593.7 1593.7 0 0 1-37 13 353 353 0 0 0-20.6 7.8 849 849 0 0 1-37.6 14l-2.1.8c-2.9 1-5 1.7-8.1 1.7l-1 2c-1.7.7-1.7.7-4 1.3-5.9 1.6-11.5 3.6-17.2 5.7a24 24 0 0 1-8.8 2l-1 2-5 2-2.3.9-2.4.8-2.7 1a4858.8 4858.8 0 0 1-12 4.4 1727.8 1727.8 0 0 0-6.4 2.3 1774.4 1774.4 0 0 1-9.2 3.3l-2.8 1-2.7 1-2.4.8c-3.2.7-5.2 0-8.1-1.5v-2h-2v-2h-2c-1.7-1.7-1.7-1.7-3.5-4-2.4-3-2.4-3-5.5-5v-2l-3-1v-2h-2v-2h-2v-2l-1.8-.8a21.7 21.7 0 0 1-5.7-4.5l-1.8-1.9-1.7-1.8a129.3 129.3 0 0 0-8.9-8.3 63 63 0 0 1-7.3-7.4 176.4 176.4 0 0 0-20.8-19.7 88.8 88.8 0 0 1-8-7.6v-2h-2l-5-6-1 664c-5.7 2.3-5.7 2.3-9.3 2.3h-62a32105.1 32105.1 0 0 1-69.7-.1H10c-6.2 0-6.2 0-7.3-1.2a58 58 0 0 1-.2-4v-27.6a20952.2 20952.2 0 0 1-.2-43 87531.5 87531.5 0 0 1 0-47.4l-.2-42.6a422579 422579 0 0 1-.5-200.8L1 1136.5v-2.1a812326.8 812326.8 0 0 1-.2-129.5v-6L.4 879.6a91455.8 91455.8 0 0 1-.1-97.9v-16a23348.6 23348.6 0 0 0-.2-41v-40a4694 4694 0 0 0 0-20.9 430.2 430.2 0 0 1 8.8-101l1-4.5c9.6-48.4 9.6-48.4 17.1-68 1.5-4 2.8-7.9 4-11.9a678.2 678.2 0 0 1 40.2-99.8 536.4 536.4 0 0 1 53-86l7.8-10.5c9.9-13.7 9.9-13.7 14-16.8 2.8-2.4 4.7-5 7-8 2.7-3.7 5.9-7 9-10.3l2-2a905.3 905.3 0 0 1 5.6-5.9l1.7-1.8 4.3-4.2h2v-2c1.4-1.6 1.4-1.6 3-3h2v-2c1.3-1.4 1.3-1.4 3.2-3l2.2-1.9 2.6-2 2.8-2.5 9.2-7.6 2.9-2.4 2.7-2.2 2.4-2c2-1.4 2-1.4 4-1.4l.7-1.8c1.5-2.6 2.8-3.6 5.3-5.2l2.7-1.8 3.3-2.2 4.1-2.8 13.9-9.2 1.9-1.3a379.8 379.8 0 0 1 44-24.7l2.5-1.2a1139.5 1139.5 0 0 1 18.6-8.8l2.4-1.1c32.4-15.1 69-24.4 104.2-30.1a81.6 81.6 0 0 0 8-1.8c5.6-1.4 11.4-2 17.1-2.7l5.8-.8c23.5-3 47-3.9 70.6-3.8zm-35.1 245.3-6.4.3a198 198 0 0 0-20.5 2.6l-3 .5c-5 .9-10 2-14.8 3.8-2.5.9-5 1.6-7.6 2.4a216 216 0 0 0-71.7 34.4 157.7 157.7 0 0 0-24.5 20.6l-9 8c-21.2 20.2-36.4 46.7-49.5 72.4l-1.5 3a312.7 312.7 0 0 0-27.5 84l-.5 2.7c-.6 3.7-.7 7.3-.6 11v15.6a11121.7 11121.7 0 0 0 0 60.3 94668.8 94668.8 0 0 1 0 173.3v191.2a8930 8930 0 0 0 9.6-3.4l2.7-1c7-2.4 13.8-5 20.6-7.8 22-9 44-17.6 66.2-26.2l3-1c4.8-2 9.7-3.8 14.5-5.7 11.2-4.3 11.2-4.3 22.3-9 9-3.9 18.4-7.3 27.7-10.8 16-6 32-12 47.8-18.7 12.4-5.2 25-10 37.6-14.8a876.7 876.7 0 0 0 42.5-16.9 411 411 0 0 1 20.9-8.2 5160 5160 0 0 0 24-9.2 4214 4214 0 0 1 5.2-2 652 652 0 0 0 27.4-11 422 422 0 0 1 8-3.4 200.7 200.7 0 0 0 52-29l8.6-6.3a229 229 0 0 0 24.6-21.6 779.4 779.4 0 0 1 4.4-4.4 217 217 0 0 0 17.5-19.7l1.6-2a250.7 250.7 0 0 0 39.6-73.8c.8-2.4 1.8-4.7 2.8-7a106 106 0 0 0 5-17.5l.8-3.5c12.3-56 9.8-117.8-11.8-171.2l-1.6-3.9a782.4 782.4 0 0 0-3-7.6 381.4 381.4 0 0 1-2.5-6.4 272.8 272.8 0 0 0-39-65l-1.8-2.6a219.7 219.7 0 0 0-16.6-18.2l-2.4-2.4-2.3-2.3-2-2.1c-1.7-1.7-1.7-1.7-3.8-1.5l-.7-1.7c-1.8-3.1-4.4-5.1-7.1-7.4l-1.6-1.4c-3-2.5-5.9-4.2-9.6-5.5v-2h-3v-2c-3.8-3-8-5.4-12-8l-2.1-1.3c-10-6.4-20-12-31-16.7l-2.1-1a404 404 0 0 0-43.8-15l-2.8-.8c-7.1-2-14.3-3.3-21.7-4.3l-3.4-.5c-14-2-27.8-2.5-42-2.5h-18c-4.2 0-8 .1-12.1 1.1zm934 3 1.9.9c9.2 4.3 17.8 9.1 26 15.1l2.6 1.8a131 131 0 0 1 48.8 86.5c3.9 32-2.7 67.8-22.8 93.6a583.5 583.5 0 0 1-10 12.3 168.2 168.2 0 0 0-2.8 3.4c-3 3.9-6.5 6.9-10.3 9.9l-2 1.4c-7.9 6-16.3 11.1-25 15.9l-2.1 1.1a92.3 92.3 0 0 1-24.4 8.2l-2.3.6a124 124 0 0 1-83.6-11.7l-2-1c-9-4.4-16.5-9.3-24-16l-2.4-2a94.3 94.3 0 0 1-25.6-32v-3h-2c-4.5-8-7.8-15.8-10.3-24.5l-.8-2.6a136 136 0 0 1-4.3-36v-2a123.5 123.5 0 0 1 8.2-45.5 123.3 123.3 0 0 1 66.4-71.6 248.6 248.6 0 0 0 4.8-2.2 118.6 118.6 0 0 1 98-.6zm27-325 2.6 2a515.6 515.6 0 0 1 7.4 6l1.9 1.5a68 68 0 0 1 13 14.5v2h2l2 5h2a142.2 142.2 0 0 1 19 106c-1.3 5.5-3 10.7-5 16l-.6 2a128 128 0 0 1-163.1 75.3c-8-3.2-15.4-6.8-22.8-11a145.2 145.2 0 0 0-6.4-3.3v-2l-5-2v-2l-1.8-.9c-2.2-1-2.2-1-5.2-3v-2l-1.8-.3c-2.8-1-4-2.3-5.8-4.6l-1.8-2a31 31 0 0 1-3.6-6.2h-2l-2.6-3.6-1.4-2.2-1.5-2.4-1.5-2.4c-3.7-5.8-3.7-5.8-5-8.5-1-2-1-2-3-3.9-5-9.8-8.7-20-11.2-30.7l-.5-2.1c-4.5-19.5-4.4-42.7.7-62.2l.5-2.2c5-18.8 15.2-35.2 27.3-50.3l2.9-4 1.3-1.5h2v-2c3.4-3.2 7.2-6 11-9l3-2.4c43.5-33.3 109.2-37.3 153-1.6zM1370.3 678l3.1-.1c6.4 0 12 .6 18.2 2.2l3.9.8a129.8 129.8 0 0 1 83 57.2c19.8 30.1 24 67.2 17 102a124.2 124.2 0 0 1-19 44h-2l-.6 2.1c-1.6 3.6-4 6.3-6.5 9.3l-1.4 1.6a100 100 0 0 1-19.7 18l-2.3 1.6a1114.3 1114.3 0 0 1-4.4 3 211.7 211.7 0 0 0-4.8 3.5c-24.9 17.3-59.8 20.7-89 15.8-25.3-5.9-46-17.7-65.2-34.9l-2-1.7a119.7 119.7 0 0 1-21-28.3v-2l-2-1a119.2 119.2 0 0 1-3.4-7.2l-1-2.2c-2.8-6.7-5-13.5-6.6-20.6l-1-3.6c-4.3-19.2-4-41.3 1-60.4l.6-2.4a151.3 151.3 0 0 1 21.4-44.6h2l.6-2.1a34 34 0 0 1 6.8-9.1l1.6-1.8a128.2 128.2 0 0 1 74.6-37.6c2.3-.3 2.3-.3 4.2-.9 4.5-1 9.2-.7 13.9-.7zm80.3 715.1a319.8 319.8 0 0 1 5 5l2.3 1.7 1.7 1.3v2h2c2.8 3 5.2 6.2 7.7 9.4a345.5 345.5 0 0 0 3.6 4.5c14 17.8 21 39 24.7 61.1l.6 3.5c3.2 27.1-3.3 55.3-15.6 79.5h-2l-1 5h-2v3h-2v3h-2l-.8 1.7c-8.3 15.6-23.4 26.7-38.2 35.3l-2.2 1.3a129.4 129.4 0 0 1-98.8 11.7 131.6 131.6 0 0 1-54-33l-1.8-1.6a123 123 0 0 1-36.3-82c-.8-28.2 3.2-57 20-80.4l2-3c19.6-29.5 48-47.1 82.2-55.8a96 96 0 0 1 7.9-1.2l3-.4c33.9-3.1 67.5 7.2 94 28.4zm4.6-333.9c4 3.3 4 3.3 5.4 5v2h2a107 107 0 0 1 4.6 5.7l1.4 1.8 2.7 3.7a254.2 254.2 0 0 0 3.4 4.5c19.4 25.2 27 59.8 23.4 91-3.9 29-14.8 53.7-34.5 75.2l-1.5 1.6a131 131 0 0 1-55.5 34.4l-3 1c-21.8 7-51.5 7.7-72.6-1.9-2.8-1.3-5.7-2.3-8.6-3.3-13-5-24-12-34.8-20.8l-2.9-2.2-2.1-1.8v-2h-2v-2l-3-1a123.1 123.1 0 0 1-33-61l-.6-2.3c-4.2-19.5-4-40.3.6-59.7l.5-2.1c5.9-24.4 20.1-49 39.5-64.9l1.8-1.6c48.3-45 120-38.4 168.8.7zm-4.6 671.9 2.5 2.1a130.3 130.3 0 0 1 43.5 123l-.4 2a147.8 147.8 0 0 1-14.6 42h-2l-1 5h-2v3h-2v3a98.9 98.9 0 0 1-3.5 3.4 107.5 107.5 0 0 1-33.8 30.5 246.8 246.8 0 0 0-4.4 2.8c-24.6 15.7-59 20.3-87.4 14.7-15-3.4-31.2-8.8-44-17.5v-2l-2-.7a34.4 34.4 0 0 1-9-6.2l-1.6-1.4a677 677 0 0 1-5.3-4.7l-1.9-1.6a736.5 736.5 0 0 1-5.3-4.8l-1.7-1.5a23 23 0 0 1-6.1-8 587.9 587.9 0 0 0-6-9l-4-6-1.5-2.4c-3.9-6-6.5-12.2-8.7-19l-1-3.2a142 142 0 0 1-2.8-72.5l.4-2c3.4-15.5 11-31.4 20.6-44h2l.9-2.3c1-2.7 1-2.7 3-5.7h2l1-1.8c1-2.2 1-2.2 3-5.2h2v-2h2v-2l4.3-3.4 7.7-6c2-1.6 2-1.6 4-1.6v-2h3v-2a160.3 160.3 0 0 1 56-19l3.2-.4c32-3.1 66.5 7.2 90.9 28.4z"/><path d="M246.6 586.1h1l1 446 5-1c-1 2-1 2-4 3.1l-3 1c-1.7-1.8-1.1-3.8-1.2-6v-17.6a11055.4 11055.4 0 0 1 0-63.8V841.6a146142.6 146142.6 0 0 0 0-133.3v-37.4a42143 42143 0 0 1 0-75.6v-2.1c0-4.9 0-4.9 1.2-7zm-54-369 2 1-37 37-2-1 6.6-7.1 1.9-2a994.8 994.8 0 0 1 5.5-5.9l1.7-1.8 4.3-4.2h2v-2c1.4-1.6 1.4-1.6 3-3h2v-2l3.9-3.5a98 98 0 0 0 6-5.5z"/></svg>
```

File: services/peacock_tv.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+id: peacock_tv
+name: Peacock TV
+rules:
+  - "||peacocktv.com^"
+  - "||peacock.com^"
```
Comment (Sergey-Lyapin) [inline services/peacock_tv.yml:5]: Quotes must be single.

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: services/peacock_tv.yml (hunk 2 above)
Comment (sandervankasteel) [reply services/peacock_tv.yml]: Done!

File: services/peacock_tv.yml (hunk 1 above)
Comment (sandervankasteel) [reply services/peacock_tv.yml]: Done

**Review by Sergey-Lyapin — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Pull request merged**